mypy/newsemanal/semanal.py

```diff
@@ -4065,7 +4206,10 @@ def add_symbol_table_node(self, name: str, symbol: SymbolTableNode,
         if (existing is not None
                 and context is not None
                 and (not isinstance(existing.node, PlaceholderNode)
-                     or isinstance(symbol.node, PlaceholderNode))):
+                     or isinstance(symbol.node, PlaceholderNode) and
+                     # Allow replacing becomes_typeinfo=False with becomes_typeinfo=True,
+                     # this can happend for type aliases and NewTypes.
```

Comment:
Typo: 'happend' -> 'happen'

Grammar nit: run-on sentence (use period instead of comma).